crates/vsock/src/vhu_vsock.rs

```diff
@@ -37,10 +37,11 @@ const TX_QUEUE_EVENT: u16 = 1;
 const EVT_QUEUE_EVENT: u16 = 2;
 
 /// Notification coming from the backend.
-pub(crate) const BACKEND_EVENT: u16 = 3;
+/// NUM_QUEUES + 1
+pub(crate) const BACKEND_EVENT: u16 = 4;
```

Comment:
Why this change?

The event queue has index 2, so we can use index 3 for `BACKEND_EVENT`, indeed we already defined `const EVT_QUEUE_EVENT: u16 = 2;`

Reply:
It's because of https://github.com/rust-vmm/vhost/blob/main/crates/vhost-user-backend/src/event_loop.rs#L117

```
        if data <= self.backend.num_queues() as u64 {
            Err(io::Error::from_raw_os_error(libc::EINVAL))
        }
```

I think it would be `data < self.backend.num_queues()`, can you confirm that?

Reply:
> It's because of https://github.com/rust-vmm/vhost/blob/main/crates/vhost-user-backend/src/event_loop.rs#L117
> 
> ```
>         if data <= self.backend.num_queues() as u64 {
>             Err(io::Error::from_raw_os_error(libc::EINVAL))
>         }
> ```
> 
> I think it would be `data < self.backend.num_queues()`, can you confirm that?

Nope, I think your changes are right, see the comment on the line before:
```
// `data` range [0...num_queues] is reserved for queues and exit event.
```

Can you also add a comment here in this change about the exit event?

Reply:
I added the comment, thanks!